manage.py

```diff
@@ -241,6 +241,7 @@ def update_schemas(processes=1):
         path = os.path.join('data', 'sql_updates', '{}.sql'.format(task))
         execute_sql_file(path)
     execute_sql_file('data/rename_temporary_views.sql')
+    execute_sql_file('data/converted_mvs/sched_a_by_size_merged.sql')
```

Comment:
See note above - or is it better due to the timing of when these scripts are run and still accounting for some dependencies?

Reply:
It is due to timing...it depends on `ofec_totals_combined_mv`